schemas/responses.properties.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "id": "https://immersive-web/gamepad-mapping/0.1.0/responses.properties.schema.json",
+    "type" : "object",
+    "definitions": {
+        "target" : {
```

Comment:
Hm... thought from chat conversation: If you allowed multiple asset formats I don't know if it's possible to guarantee compatible node paths between all of them. You'd probably need another level of indirection where values like this indexed into an array of asset-specific paths. (Bleh.)

Reply:
Bleh indeed.

Reply:
Filed #6 to address this